game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java

```diff
@@ -208,8 +236,50 @@ private void initInterviewStage() {
 
         //Add response box to screen
         interviewStage.addActor(responseBoxTable);
+        interviewStage.addActor(scoreLabel);
     }
 
+    /**
+     * This method is called when the player has won the game
+     */
+    private void winGame()
+    {
+        game.guiController.narratorScreen.setButton("Return to Office", new Runnable() {
+            @Override
+            public void run() {
+                Gdx.app.exit();
+            }
+        });
+
+        String room = "";
+        String weapon = "";
+
+        //Get the murder room name and the murder weapon
+        for (Room r : game.gameSnapshot.map.getRooms()) {
+            if (r.isMurderRoom()) {
+                room = r.getName();
+            }
+
+            for (Clue c : r.getClues()) {
```

Comment:
Is there not a nicer way of getting the means clue? Nested for loops are never a good thing. Also I don't understand why there are two for loops nested inside here, when they're both assigning to the same variable